src/lib/components/ButtonGroup/__tests__/ButtonGroup.test.jsx

```diff
@@ -38,27 +32,27 @@ describe('rendering', () => {
     ...idPropTest,
     [
       { priority: 'filled' },
-      (rootElement) => expect(within(rootElement).getByRole('button')).toHaveClass('rootPriorityFilled'),
+      (rootElement) => expect(within(rootElement).getByRole('button')).toHaveClass('isRootPriorityFilled'),
     ],
     [
       { priority: 'outline' },
-      (rootElement) => expect(within(rootElement).getByRole('button')).toHaveClass('rootPriorityOutline'),
+      (rootElement) => expect(within(rootElement).getByRole('button')).toHaveClass('isRootPriorityOutline'),
     ],
     [
       { priority: 'flat' },
-      (rootElement) => expect(within(rootElement).getByRole('button')).toHaveClass('rootPriorityFlat'),
+      (rootElement) => expect(within(rootElement).getByRole('button')).toHaveClass('isRootPriorityFlat'),
     ],
     [
       { size: 'small' },
```

Comment:
What about replacing `size`  tests with shared prop test as you did with block?

Only Modal has different set, but other components share the same set of size options.

What do you think @adamkudrna ?

Reply:
Those components that need a simple `rootElement` test do use a common `sizePropTest`. Others that are described differently have their own tests.

Reply:
Did you mean `size` and `priority` prop tests? I don't see how it would benefit us in these cases. Configurable `sizePropTest` would be done only to add its support for `Modal` and `ButtonGroup`. I can't think of any other way to implement it than just to move the test descriptions into an if statement inside a function returning the right tests. I believe it would be pretty much the same amount of code in exchange for it being less readable. It's similar for `priorityPropTest`.
I made `Toolbar` related tests configurable, though. It came in handy there!